.php_cs.cache

```diff
@@ -0,0 +1 @@
_to_null_coalescing":true,"trailing_comma_in_multiline_array":true,"trim_array_spaces":true,"unary_operator_spaces":true,"void_return":true,"whitespace_after_comma_in_array":true,"PhpCsFixerCustomFixers\/comment_surrounded_by_spaces":true,"PhpCsFixerCustomFixers\/no_unneeded_concatenation":true,"PhpCsFixerCustomFixers\/no_duplicated_imports":true,"PhpCsFixerCustomFixers\/internal_class_casing":true,"PhpCsFixerCustomFixers\/no_php_storm_generated_comment":true,"PhpCsFixerCustomFixers\/no_useless_comment":true,"PhpCsFixerCustomFixers\/no_useless_sprintf":true,"PhpCsFixerCustomFixers\/single_space_before_statement":true,"PhpCsFixerCustomFixers\/no_useless_doctrine_repository_comment":true,"PhpCsFixerCustomFixers\/no_doctrine_migrations_generated_comment":true},"hashes":{"tests\/FixtureHelper.php":3919032672,"tests\/bootstrap.php":1986882695,"tests\/Writer\/ProductsTest.php":769163791,"tests\/Writer\/CategoriesTest.php":4130206537,"tests\/Writer\/Xml\/XmlFormatterTest.php":1512603357,"tests\/Writer\/Xml\/XmlWriterTest.php":128684509,"tests\/Writer\/VariantsTest.php":3001483826,"src\/Parser.php":2060948395,"src\/XmlException.php":91947600,"src\/Parser\/ProductSimpleNodeParser.php":1552717498,"src\/Parser\/CommonDetailsNodeParserTrait.php":3189337047,"src\/Parser\/VariationNodeParser.php":3585376264,"src\/Parser\/CategoryNodeParser.php":1737508244,"src\/Parser\/ProductNodeParser.php":2144264684,"src\/Parser\/BundleSimpleNodeParser.php":32103431,"src\/Parser\/SetNodeParser.php":267598727,"src\/Parser\/CategorySimpleNodeParser.php":3707588571,"src\/Parser\/SetSimpleNodeParser.php":2638146880,"src\/Parser\/AssignmentNodeParser.php":1020741910,"src\/Parser\/BundleNodeParser.php":1218240582,"src\/Parser\/VariationSimpleNodeParser.php":1632066821,"src\/Parser\/NodeParserInterface.php":3638536871,"src\/Writer\/Entity\/Category.php":727697203,"src\/Writer\/Entity\/DeletedProduct.php":702757992,"src\/Writer\/Entity\/Product.php":1307386162,"src\/Writer\/Entity\/CustomAttribute.php":3446753471,"src\/Writer\/Entity\/Variant.php":3652802165,"src\/Writer\/Entity\/WriteableEntityInteface.php":1550672502,"src\/Writer\/Entity\/DeletedCategory.php":4044479396,"src\/Writer\/Entity\/DeletedAssignment.php":42814339,"src\/Writer\/Entity\/Assignment.php":2107866917,"src\/Writer\/Xml\/XmlWriter.php":4135224338,"src\/Writer\/Xml\/XmlFormatterException.php":3166338478,"src\/Writer\/Xml\/NotEmptyWriter.php":3281015189,"src\/Writer\/Xml\/XmlFormatter.php":4120015297,"src\/Writer\/EntityWriter\/CategoryXmlWriter.php":680262121,"src\/Writer\/EntityWriter\/AssignmentXmlWriter.php":1901939966,"src\/Writer\/EntityWriter\/DeletedProductXmlWriter.php":3432497426,"src\/Writer\/EntityWriter\/DeletedAssignmentXmlWriter.php":3859287862,"src\/Writer\/EntityWriter\/ProductXmlWriter.php":1462905887,"src\/Writer\/EntityWriter\/CustomAttributeWriter.php":2885219160,"src\/Writer\/EntityWriter\/DeletedCategoryXmlWriter.php":1014756056,"src\/Writer\/EntityWriter\/VariantXmlWriter.php":4125945097}}
```

Comment:
@liamkeily You've committed a cache file here.

Reply:
@ziadoz see https://github.com/fusionspim/php-demandware-xml/pull/101